packages/pyright-internal/src/languageService/completionProvider.ts

```diff
@@ -926,15 +959,21 @@ export class CompletionProvider {
             return sb;
         }
 
-        const parameters = getParameters(declaredType, decl);
+        const parameters = getParameters();
         if (decl.node.name.value !== '__init__') {
             sb += 'return ';
         }
 
+        if (isProperty) {
+            // If it's a property, it's not clear what to provide in the
+            // implementation. Assume that the user will fill in the expression.
+            return sb;
```

Comment:
It'd actually be `return super().prop`, if we did want to generate something.

Reply:
Or did you see an ambiguity in this case?

Reply:
Yeah, that's reasonable.